tools/devops/device-tests/templates/device-tests.yml

```diff
@@ -246,24 +246,24 @@ steps:
 - bash: |
     set -x
     set -e
-    echo 'Use xamarin storage $USE_XAMARIN_STORAGE'
+    P=$(echo $XM_STORAGE_PATH | tr -d \'\")
```

Comment:
Can we add a small note about why this is necessary + what it does? Hopefully will be able to remove it in the future.

Reply:
I'll see if the comment works, but is in the commit, git blame will show the info